Web/Edubase.Web.UI/Areas/Groups/Views/Group/Details.cshtml

```diff
@@ -104,15 +104,15 @@
         {
             if (Model.IsClosedInError)
             {
-                <div class="gias-establishment-closed-message">
+                <div id="gias-establishment-closed-message" class="gias-establishment-closed-message">
```

Comment:
should this be id="gias-group-closed-message"

Reply:
The message is related to the closing the establishment that is linked to a group not related to the closing the group